lib/linter.js

```diff
@@ -0,0 +1,29 @@
+const _ = require('lodash');
+const Result = require('./result-model');
+
+class Linter {
+  constructor(options) {
+    this.matcher = options.matcher;
+    this.pathParser = options.pathParser || _.identity;
+
+    _.bindAll(this, ['filePathTest', 'lint']);
```

Comment:
Can you give an example? As they are class methods, when you create an instance of `Linter` these methods are bound to this instance right?